### PR TITLE
Add notifications on user chip and achievements

### PR DESCRIPTION
**Pull request by AlexandreSi**
Display notification badge on homepage's user chip
image

Display notification badge on achievement line (and disappear after 5s)
image

Checked on Safari and Firefox

### CODE REVIEW

File: newIDE/app/src/MainFrame/EditorContainers/HomePage/index.js [hunk 1]
```diff
@@ -41,6 +41,7 @@ import PreferencesContext from '../../Preferences/PreferencesContext';
 import { type FileMetadataAndStorageProviderName } from '../../../ProjectsStorage';
 import generateName from '../../../Utils/ProjectNameGenerator';
 import { sendTutorialOpened } from '../../../Utils/Analytics/EventSender';
+import { arePendingNotifications } from '../../../Utils/Notification';
```
Comment (4ian) [inline newIDE/app/src/MainFrame/EditorContainers/HomePage/index.js:44]: nit: `hasPendingNotifications` or `areNotificationsPending`?

File: newIDE/app/src/Profile/ProfileDetails.js [hunk 2]
```diff
@@ -91,7 +91,7 @@ const ProfileDetails = ({
             />
           </Line>
           {isAuthenticatedUserProfile && (
-            <ResponsiveLineStackLayout justifyContent="flex-end">
+            <ResponsiveLineStackLayout justifyContent="flex-end" noColumnMargin>
```
Comment (4ian) [inline newIDE/app/src/Profile/ProfileDetails.js:94]: 👍

File: newIDE/app/src/stories/componentStories/UserChip/UserChip.stories.js [hunk 3]
```diff
@@ -6,22 +6,43 @@ import { action } from '@storybook/addon-actions';
...
+  argTypes: {
+    withNotifications: {
+      name: 'With notifications',
+      type: { name: 'boolean', required: true },
+      defaultValue: false,
+    },
+    profile: {
+      name: 'User profile',
+      control: { type: 'radio' },
+      options: ['Signed in', 'Unsigned'],
```
Comment (4ian) [inline newIDE/app/src/stories/componentStories/UserChip/UserChip.stories.js:26]: uberpitcking: not sure if Unsigned is proper english, maybe "Not signed in" (https://english.stackexchange.com/questions/251800/a-word-for-a-non-logged-in-user)
(anonymous could work too)

**Review by 4ian — APPROVED** [review]

File: newIDE/app/src/MainFrame/EditorContainers/HomePage/index.js (hunk 1 above)
Comment (AlexandreSi) [reply newIDE/app/src/MainFrame/EditorContainers/HomePage/index.js:44]: Yeah, better

Comment (AlexandreSi) [reply newIDE/app/src/MainFrame/EditorContainers/HomePage/index.js:44]: (The first 😅 )

**Pull request merged**